docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       DB_NAME: testnet_api
       DB_USER: testnet_api
       DB_PASSWORD: testnet_api
+      DISCOVERY_VERIFY_IDENTITY: "false"
```

Comment:
As we talked with @vkuznecovas this flag is legacy and running node locally is not needed so I disabled it for local environment.